### PR TITLE
added encrypted key to struct

### PR DESCRIPTION
**Pull request by gioelecerati**

### CODE REVIEW

File: api.go [hunk 1]
```diff
@@ -270,8 +270,9 @@ type (
 	UploadTaskParams struct {
 		URL        string `json:"url,omitempty"`
 		Encryption struct {
-			Key       string `json:"key"`
-			Algorithm string `json:"algorithm,omitempty"`
+			EncryptedKey string `json:"encryptedKey,omitempty"`
```
Comment (victorges) [inline api.go:273]: Should we have the public key used for encryption as well? Would be future-proof and safer

File: api.go [hunk 2]
```diff
@@ -270,8 +270,9 @@ type (
...
+			Key          string `json:"key,omitempty"`
```
Comment (victorges) [inline api.go:274]: Can you add a comment saying why this is still here? We should not have both/never pass unencrypted keys around, so it needs at least a TODO to remove it once we kill the other experiment.

**Review by victorges — APPROVED** [review]

File: api.go (hunk 2 above)
Comment (leszko) [reply api.go:274]: To me, it's also not clear why we need both encrypted and decrypted keys here. Looking at the design [VOD Access Control V2](https://www.notion.so/livepeer/VOD-Access-Control-V2-Technical-doc-240d77ebac824db998172c4e00542e4b), I don't see any place where Catalyst would need to fetch encrypted and descrypted keys from Studio. Could you maybe update the design or add some comments here?

Comment (victorges) [reply api.go:274]: I think this was a reminiscent of an old experiment I had implemented where the key came unencrypted in this field. I don't think it should stay around, so I'd rather remove it and avoid the confusion in the schema.

**Pull request merged**